src/coreclr/tools/Common/Internal/Text/Utf8StringBuilder.cs

```diff
@@ -84,9 +76,7 @@ public string ToString(int start)
 
         public Utf8String ToUtf8String()
         {
-            var ret = new byte[_length];
-            Buffer.BlockCopy(_buffer, 0, ret, 0, _length);
-            return new Utf8String(ret);
+            return new Utf8String(_buffer.AsSpan(0, _length).ToArray());
```

Comment:
```suggestion
            return new Utf8String(AsSpan().ToArray());
```

Reply:
This suggestion does not look right. The existing code makes the string from the filled-in portion of the buffer that is the correct behavior.

Reply:
AsSpan is defined as:
```cs
public ReadOnlySpan<byte> AsSpan() => _buffer.AsSpan(0, _length);
```
are they not equivalent?

Reply:
Ah, ok. I have misread this as calling `_buffer.AsSpan()`. Looks good.